src/core/PasswordGenerator.cpp

```diff
@@ -133,9 +137,26 @@ bool PasswordGenerator::isValid() const
         return false;
     }
 
+    if(passwordChars().isEmpty())
+      return false;
```

Comment:
Coding style: we always use braces even if it's a single line statement. Space after `if`.